src/atomicui/organisms/RouteBox/RouteBox.tsx

```diff
@@ -268,7 +268,8 @@ const RouteBox: React.FC<RouteBoxProps> = ({ mapRef, setShowRouteBox, isSideMenu
 								: setSuggestions({ ...suggestions, to: sg });
 						},
 						TriggeredByEnum.ROUTE_MODULE,
-						action
+						action,
+						false
```

Comment:
Same as comment above. What is the purpose of this change?

Reply:
removed it, set a default value for that parameter